app/models/ems_refresh.rb

```diff
@@ -55,6 +55,19 @@ def self.queue_refresh(target, id = nil, sync = false)
     end
   end
 
+  def self.queue_refresh_new_target(event, ems)
+    MiqQueue.put_or_update(
+      :queue_name  => MiqEmsRefreshWorker.queue_name_for_ems(ems),
+      :class_name  => name,
+      :method_name => 'refresh_new',
```

Comment:
Probably nitpicking but this should be `refresh_new_target` to match the rest of the method names